_monitoring-your-cluster/pa/reference.md

```diff
@@ -821,27 +821,173 @@ The following metrics are relevant to the cluster as a whole and do not require
    </tbody>
 </table>
 
+## Relevant dimensions: `NodeID`, `searchbp_mode`
+
+<table>
+  <thead style="text-align: left">
+  <tr>
+    <th>Metric</th>
+    <th>Description</th>
+  </tr>
+ </thead>
+ <tbody> 
+    <tr>
+    <td>searchbp_shard_stats_cancellationCount
+    </td>
+    <td>The number of tasks marked for cancellation on the shard task.
+    </td>
+  </tr>
+  <tr>
+    <td>searchbp_shard_stats_limitReachedCount
+    </td>
+    <td>The number of times when the cancellable task total exceeded the set cancellation threshold on the shard task.
+    </td>
+  </tr>
+  <tr>
+    <td>searchbp_shard_stats_resource_heap_usage_cancellationCount
+    </td>
+    <td>The number of tasks marked for cancellation because of excessive heap usage since the node last restarted on the shard task.
+    </td>
+  </tr>
+  <tr>
+    <td>searchbp_shard_stats_resource_heap_usage_currentMax
+    </td>
+    <td>The maximum heap usage for tasks currently running on the shard task.
+    </td>
+  </tr>
+  <tr>
+    <td>searchbp_shard_stats_resource_heap_usage_rollingAvg
+    </td>
+    <td> The rolling average heap usage for the _n_ most recent tasks on the shard task. The default value for _n_ is 100.
+    </td>
+  </tr>
+  <tr>
+    <td>searchbp_shard_stats_resource_cpu_usage_cancellationCount
```

Comment:
@kaushalmahi12 : We follow the `CamelCase` naming convention for metric naming. Can you provide the same for these metrics.

Reply:
> @kaushalmahi12 : We follow the `CamelCase` naming convention for metric naming. Can you provide the same for these metrics.

These look right to me. Can you please indicate which ones are incorrect?

Reply:
For reference https://github.com/opensearch-project/documentation-website/blob/9b354eb15e519b33ea702f8e7cc67bde1aa4fb6e/_monitoring-your-cluster/pa/reference.md?plain=1#L30 

`CPU_Utilization` - The Metric name begins with Uppercase letter, and uses `_` .  Following on same line, `searchbp_shard_stats_resource_cpu_usage_cancellationCount` will be `SearchBP_Shard_Stats_Resource_CPU_Usage_CancellationCount`.

Reply:
Thanks, @khushbr ! @Naarcha-AWS will do the doc review and merge.